worker/src/core/loaders/message-worker/email.class.ts

```diff
@@ -103,10 +147,18 @@ class Email {
           subject: string
           hydratedBody: string
         }) => {
+          const unsubUrl = this.generateUnsubLink(
```

Comment:
Do we want to enforce some form of encoding here, like base 64?
`PUT http://localhost:4000/v1/unsubscribe/40/abc@open.gov.sg` feels weird, because it seems like the `:recipient` part of the link (`abc@open.gov.sg`) could potentially cause problems if ever a recipient contained a slash `/`, or unexpected characters. https://stackoverflow.com/a/2071250

Reply:
Decided to `encodeURIComponent` the recipient on the frontend before calling `PUT`. Didn't go with the base64 approach because `/` is [part of the set of characters](https://en.wikipedia.org/wiki/Base64) also.